scripts/components/keepfollowing.lua

```diff
@@ -79,26 +125,6 @@ end)
 --- Helpers
 -- @section helpers
 
-local function HidePlayer(inst)
```

Comment:
After rebasing, I've decided not to fully resolve conflicts on my own and just accept your variant. It seems that your branch is missing the merged #8. It could be either that you've created a branch from an outdated `develop` or from `main`.